Cows-HacknSlash/Assets/Scripts/PlayerController/CharController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class CharController : MonoBehaviour {
+
+	private NavMeshAgent navMeshAgent;
+	private bool running;
+	[SerializeField]
+	private LayerMask layerMask;
+	
+	
+	[Header("Misc")]
+	public GameObject cursorClick;
+
+	public Vector3 cursorOffset;
+	void Start () {
+		navMeshAgent = GetComponent<NavMeshAgent>();
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+		Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
+
+		RaycastHit hit;
+
+
+		Vector3 pointToLook = ray.GetPoint(50f);
+		Debug.DrawLine(ray.origin, pointToLook, Color.blue);
+		
+		if(Input.GetMouseButton(0))
+		{
+			if(Physics.Raycast(ray, out hit, 100, layerMask))
+			{
+				navMeshAgent.destination = hit.point;
+				
```

Comment:
No need for this whiteline inside this if statement, also try to add spaces between the if and the bracket '('